piksi_tools/console/settings_view.py

```diff
@@ -862,39 +761,26 @@ def __init__(self,
                  gui_mode=True,
                  skip=False):
         super(SettingsView, self).__init__()
-        self.ordering_counter = 0
+        self.s_api = Settings(1234, link)
```

Comment:
Suggest `s_api` -> `settings_api` or `libsettings`, or `settings_instance`.

This used to default to SENDER_ID from `sbp.msg` from what I can tell (https://github.com/swift-nav/libsbp/blob/master/python/sbp/settings.py#L79).  Should we do the same here?

Reply:
Renamed to `settings_api`. Need to verify the ID issue.